voluncheer/voluncheer/settings.py

```diff
@@ -172,10 +177,11 @@
 EMAIL_BACKEND = "django_ses.SESBackend"
 AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY")
 AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
-AWS_SES_REGION_NAME = "us-west-2"  # (ex: us-east-2)
+AWS_SES_REGION_NAME = "us-east-2"  # (ex: us-east-2)
 AWS_SES_REGION_ENDPOINT = (
-    "email.us-west-2.amazonaws.com"  # (ex: email.us-east-2.amazonaws.com)
+    "email.us-east-2.amazonaws.com"  # (ex: email.us-east-2.amazonaws.com)
 )
+DEFAULT_FROM_EMAIL = "noreply.voluncheer@gmail.com"
```

Comment:
Do we/you actually have this email address?

Reply:
Yes for now I suppose. If we stick to the AWS email service.